crates/biome_cli/src/commands/mod.rs

```diff
@@ -207,6 +203,14 @@ pub enum BiomeCommand {
         #[bpaf(long("apply-unsafe"), switch, hide_usage)]
         apply_unsafe: bool,
 
+        /// Bulk fix diagnostics with suppression comments if the language supports it.
```

Comment:
```suggestion
        /// Fixes lint rule violations with a comment a suppression instead of using a rule code action (fix)
```

The "if the language supports it" is our technical debt, meaning that our stable linters must always provide a code action; if not, it's a bug